app/models/terms_of_service.rb

```diff
@@ -16,6 +16,7 @@
 class TermsOfService < ApplicationRecord
   scope :published, -> { where.not(published_at: nil).order(Arel.sql('coalesce(effective_date, published_at) DESC')) }
   scope :live, -> { published.where('effective_date IS NULL OR effective_date < now()').limit(1) }
+  scope :upcoming, -> { published.reorder(effective_date: :asc).where('effective_date IS NOT NULL AND effective_date > now()').limit(1) }
```

Comment:
A good follow-up to this PR would be a small refactor of these scopes to keep each one to do fewer things, which would probably simplify some of the combining/reordering/usage of them. I realize this PR is not creating the situation though, so prob fine as-is here.